internal/campaign/campaign.go

```diff
@@ -170,3 +190,75 @@ func readPrDescriptionFile(filename string) (string, string, error) {
 
 	return prTitle, strings.Join(prBodyLines, "\n"), nil
 }
+
+func CreateInitialFiles(campaignName string) error {
+	data := TemplateVariables{
+		CampaignName: campaignName,
+	}
+	files := map[string]string{
+		".gitignore": gitignoreTemplate,
+		".turbolift": turboliftTemplate,
+		"README.md":  readmeTemplate,
+		"repos.txt":  reposTemplate,
+	}
+	for filename, templateFile := range files {
+		err := applyTemplate(filepath.Join(campaignName, filename), templateFile, data)
+		if err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+// Applies a given template and data to produce a file with the outputFilename
+func applyTemplate(outputFilename string, templateContent string, data interface{}) error {
+	file, err := os.Create(outputFilename)
+	if err != nil {
+		return fmt.Errorf("unable to open file for output: %w", err)
+	}
+
+	defer file.Close()
+
+	parsedTemplate, err := template.New("").Parse(templateContent)
+
+	if err != nil {
+		return fmt.Errorf("unable to parse template: %w", err)
+	}
+
+	err = parsedTemplate.Execute(file, data)
+
+	if err != nil {
+		return fmt.Errorf("unable to write templated file: %w", err)
+	}
+	return nil
+}
+
+func ApplyReadMeTemplate(filename string, dirName string) error {
+	data := TemplateVariables{
+		CampaignName: dirName,
+	}
+	err := applyTemplate(filename, readmeTemplate, data)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+func PrDescriptionUnchanged(dir *Campaign) (bool, error) {
+	tempPrDescriptionFile, err := os.Create("turbolift-temp-pr-description-*")
```

Comment:
Could we make this work without creating a new file? e.g. if `applyTemplate` accepted an `io.Writer` or a buffer instead of a file, and file creation were moved out of it. 

Then this function could avoid creating a new file, and instead just read the real file and compare its string contents...